spec/controllers/glossaries_controller_spec.rb

```diff
@@ -167,4 +149,26 @@
     end
   end
 
+  describe "duplicate" do
+    it "duplicates the requested glossary" do
+      expect {
+        post :duplicate, {:id => glossary.id}
+      }.to change(Glossary, :count).by(1)
+      expect(assigns(:new_glossary).id).not_to eq(glossary.id)
+    end
+
+    it "redirects to the glossary edit for duplicate" do
+      post :duplicate, {:id => glossary.id}
+      expect(response).to redirect_to(edit_glossary_url(assigns(:new_glossary)))
+    end
+  end
+
+  describe "export" do
+    it "exports the requested glossary" do
+      post :export, {:id => glossary.id}
```

Comment:
switch to get

Reply:
Fixed in https://github.com/concord-consortium/lara/pull/931